commit db1bf17777068a34e7586236efefb74a42460932
Author: Volker Krause <vkrause@kde.org>
Date:   Sun Apr 11 10:54:57 2021 +0000

    Allow to disable building manual tests
    
    This is consistent with all the other test programs, and helps in
    environments where building regular executables doesn't work out of
    the box, such as the Android x86 32bit environment I have here.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 377ea0d0..0ad19b22 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,6 +56,7 @@ option(BUILD_GTK_TESTS "Whether to compile the GTK+ test programs." ON)
 option(BUILD_QT5_TESTS "Whether to compile the Qt5 test programs." ON)
 option(BUILD_QT6_TESTS "Whether to compile the Qt6 test programs." ON)
 option(BUILD_CPP_TESTS "Whether to compile the CPP test programs." ON)
+option(BUILD_MANUAL_TESTS "Whether to compile manual test programs." ON)
 option(ENABLE_SPLASH "Build the Splash graphics backend." ON)
 option(ENABLE_UTILS "Compile poppler command line utils." ON)
 option(ENABLE_CPP "Compile poppler cpp wrapper." ON)
@@ -752,7 +753,9 @@ endif()
 if(ENABLE_GLIB)
   add_subdirectory(glib)
 endif()
-add_subdirectory(test)
+if (BUILD_MANUAL_TESTS)
+  add_subdirectory(test)
+endif()
 if(ENABLE_QT5)
   add_subdirectory(qt5)
 endif()
